uportal-war/src/main/webapp/WEB-INF/flows/edit-account/editLocalAccount.jsp

```diff
@@ -62,44 +62,73 @@
             <!-- Portlet Section -->
             <div class="portlet-section" role="region">
                 <div class="content">
-
                     <table class="portlet-table table table-hover">
```

Comment:
I'm wavering on this.
`<table>` tags should have `<caption>` tags that describe their content.
However, these tables are used as layout that don't have their own content.
http://webaim.org/techniques/tables/data#caption

I would lean toward using the `<fieldset>` and `<legend>` tags.
But that would be disruptive to the layout, which would need to be refactored to use the bootstrap grid system.
https://www.w3.org/WAI/tutorials/forms/grouping/#associating-related-controls-with-fieldset

Thoughts on which technique would work better? :thought_balloon: